README.md

```diff
@@ -18,15 +18,11 @@ Marten also provides .NET developers with an ACID-compliant event store with use
 
 Before getting started you will need the following in your environment:
 
-**1. .NET Core SDK 3.1 (or higher)**
+**1. .NET Core SDK 5.0 (or higher)**
```

Comment:
Should this include .NET Core SDK 3.1 as well?

Reply:
Not required, .NET Core SDKs are backwards compatible with lower targets.

Reply:
I think the only exception to that would be for pointing the tests at the specific runtime version.

Reply:
we can leave it as `.NET Core SDK 3.1 (or higher)`, this also means that it includes net5. thoughts?

Reply:
I adjusted the wording to be more accurate